backend/data/scrapers/programs_formatting.py

```diff
@@ -33,14 +33,22 @@ def format_prg_data() -> None:
     # Initialise formatted data
     programs_formatted: dict[str, dict] = {}
     for program in raw_content:
+        # if program['studyLevelValue'] != 'ugrd':
+        #     continue
```

Comment:
comment?

Reply:
k i ll get rid of it cos i have the undergrad check when scraping